README.md

```diff
@@ -18,133 +18,230 @@ lpthread               | LGPL
+$ make
+$ make install
+```
+
+**4.** Create symlinks to folders in scripts repository
+```
+$ ln -s ../sdl_atf_test_scripts/files
+$ ln -s ../sdl_atf_test_scripts/test_sets
+$ ln -s ../sdl_atf_test_scripts/test_scripts
+$ ln -s ../sdl_atf_test_scripts/user_modules
+```
+
+**5.** Install dependencies for local parallel mode
+
+Steps below are required only in case if local parallel mode is going to be used.
+- Install Docker, e.g. [how-to](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-18-04)
+- Install additional tools:
+```
+$ sudo apt-get install screen rsync
+```
+- Create docker image
+```
+$ cd atf_parallels/docker
+$ ./build.sh <ubuntu_version>
+```
+<b>Note:</b> accepted values are `16` and `18` (will be processed as `16.04` and `18.04` correspondingly).
+If version is not specified `18` will be used by default.
+
+## Settings
+
+### Defaults
 
-## Configuration of ATF
-ATF configuration is setting up in `modules/configuration` folder.
-- `base_config.lua` : base configuration parameters (reporting, paths to SDL e.t.c)
-- `connection_config.lua` : configuration parameters related to all connections (mobile, hmi, remote)
-- `security_config.lua` : configuration parameters related to security layer of connection
-- `app_config.lua` : predefined applications parameters
-Each folder in this folder represents values of `--config` option for ATF run: `local`, `remote_linux`, `remote_qnx`
-They can override one or more described configuration files.
+By default ATF reads the following parameters from `Default` configuration (`./modules/configuration/`).
 
-## Run:
+Config File           | Config Parameter                | Cmd Argument | Description
+----------------------|---------------------------------|--------------|-----------------------------------------
+base_config.lua       | config.SDL                      |              | SDL binary name
+base_config.lua       | config.reportPath               | --report     | Path to reports and logs
+base_config.lua       | config.pathToSDL                | --sdl-core   | Path to SDL binaries
+base_config.lua       | config.pathToSDLInterfaces      | --sdl-api    | Path to SDL APIs
+connection_config.lua | config.remoteConnection.enabled |              | Defines if remote connection is enabled
 
-1. Copy `RemoteTestingAdapterServer` folder to SDL host and run `RemoteTestingAdapterServer` on that host
+### Priorities
+
+1. Command line argument
+2. Specific config
+3. Base config
+
+Where 1 is max and 3 is min priority.
+
+This means if some particular option is defined in all 3 places ATF will use the one from highest priority place.
+
+E.g. if path to SDL binaries is defined through `--sdl-core` command line argument (1), in specific config (2) and also in base config (3), ATF will use the value from command line argument.
+
+## Run
 
-2. Start ATF script:
 ```
-./start.sh [--config=<config_name>] [options] [script file name]
+./start.sh TEST [OPTIONS]...
```

Comment:
`TEST` is variable that usually different and it works it we will swap :
`./start.sh [OPTIONS] TEST`